tests/Statable/StatableTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Sebdesign\SM\Test\Statable;
+
+use Sebdesign\SM\Test\TestCase;
+use SM\StateMachine\StateMachine;
+use Sebdesign\SM\Services\StateHistoryManager;
+
+class StatableTest extends TestCase
+{
+    /** @var StatableArticle */
+    public $article;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->app['config']->set('state-machine.graphA.class', StatableArticle::class);
+        $this->app['config']->set('state-machine.graphA.callbacks.after.history.do', [StateHistoryManager::class, 'storeHistory']);
+
+        $this->article = new StatableArticle('new', 2);
+    }
+
+    /**
+     * @test
+     */
+    public function testInstantiateSM()
```

Comment:
The `test` prefix is not needed in the method name because of the `@test` annotation.
Also I think I prefer snake_case for test methods: `it_returns_the_current_state`